lib/matplotlib/widgets.py

```diff
@@ -1002,43 +1002,24 @@ def __init__(self, ax, labels, actives=None):
         if actives is None:
             actives = [False] * len(labels)
 
-        if len(labels) > 1:
-            dy = 1. / (len(labels) + 1)
-            ys = np.linspace(1 - dy, dy, len(labels))
-        else:
-            dy = 0.25
-            ys = [0.5]
-
-        axcolor = ax.get_facecolor()
-
-        self.labels = []
-        self.lines = []
-        self.rectangles = []
-
-        lineparams = {'color': 'k', 'linewidth': 1.25,
-                      'transform': ax.transAxes, 'solid_capstyle': 'butt'}
-        for y, label, active in zip(ys, labels, actives):
-            t = ax.text(0.25, y, label, transform=ax.transAxes,
-                        horizontalalignment='left',
-                        verticalalignment='center')
-
-            w, h = dy / 2, dy / 2
-            x, y = 0.05, y - h / 2
-
-            p = Rectangle(xy=(x, y), width=w, height=h, edgecolor='black',
-                          facecolor=axcolor, transform=ax.transAxes)
+        ys = np.linspace(1, 0, len(labels)+2)[1:-1]
+        text_size = mpl.rcParams["font.size"] / 2
```

Comment:
This seems unused? (Otherwise I would have asked why it is halved?)

Reply:
That's a mistake. It's supposed to be used for marker size. As to why it is halved, it's because radio buttons are also half the text size, that's why. I think it's arbitrary.